### PR TITLE
Mallard demographic model

### PR DESCRIPTION
**Pull request by petrelharp**
Here's a demographic model for AnaPla from Lavretsky et al.!

### CODE REVIEW

**Comment by jeromekelleher** [comment]
Can you rebase please @petrelharp, #826 has been merged.

**Comment by petrelharp** [comment]
This isn't ready for review because we don't actually have the correct ancestral pop size, but it should be a good example for how to do a demographic model PR.

**Comment by petrelharp** [comment]
I got the missing number from the authors - I think this is ready to go.

**Review by jeromekelleher — APPROVED** [review]
LGTM!

File: stdpopsim/catalog/AnaPla/demographic_models.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            stdpopsim.Citation(
+                author="Lavretsky et al.",
+                year=2019,
+                doi="https://doi.org/10.1111/mec.15343",
+                reasons={stdpopsim.CiteReason.DEM_MODEL},
+            )
+        ],
+        generation_time=4,
+        model=model,
+        # mutation_rate=4.83e-9,
```
Comment (apragsdale) [inline stdpopsim/catalog/AnaPla/demographic_models.py:65]: This could be uncommented now, I think - and will be checked in QC!

**Review by apragsdale — COMMENTED** [review]
Looks great! Just the commented out mutation rate, as the PR to handle that has been merged

**Comment by petrelharp** [comment]
Argh, I forgot to update the parameter table.

**Pull request merged**